<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CarGroupBox</class>
 <widget class="QWidget" name="CarGroupBox">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>761</width>
    <height>80</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Car ID:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="idLabel">
     <property name="text">
      <string>0</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Mode:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Filter:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Motion Model:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="3">
    <widget class="QComboBox" name="motionModelComboBox">
     <property name="enabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="6">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Controller:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="7">
    <widget class="QComboBox" name="controllerComboBox">
     <property name="enabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QRadioButton" name="assistedRadioButton">
     <property name="text">
      <string>Assisted</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QRadioButton" name="autoRadioButton">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>Auto</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <property name="autoExclusive">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QRadioButton" name="manualRadioButton">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>Manual</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
     <property name="autoExclusive">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="7">
    <widget class="QRadioButton" name="notConnectedRadioButton">
     <property name="text">
      <string>Not Connected</string>
     </property>
    </widget>
   </item>
   <item row="3" column="5">
    <widget class="QComboBox" name="handControllercomboBox"/>
   </item>
   <item row="3" column="1">
    <widget class="QComboBox" name="filterComboBox">
     <property name="enabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="4">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>HandController:</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
